images/debian.yaml

```diff
@@ -831,6 +831,13 @@ files:
 
  - path: /etc/machine-id
    generator: remove
+   types:
+    - container
+
+ - path: /etc/machine-id
+   generator: dump
+   types:
+    - vm
```

Comment:
Why the differing logic?

Reply:
Misread your previous comment. Fixed it.